docs/output.md

```diff
@@ -108,44 +138,93 @@ The [bigWig](https://genome.ucsc.edu/goldenpath/help/bigWig.html) format is in a
 
 The creation of these files can be bypassed by setting the parameters `--skip_bigwig`/`--skip_bigbed`.
 
-*Output directories*:
+## Transcript Reconstruction and Quantification
 
-* `<ALIGNER>/bigwig/`  
-  Per-sample `*.bigWig` files.
-* `<ALIGNER>/bigbed/`  
-  Per-sample `*.bigBed` files.
+<details markdown="1">
+<summary>Output files</summary>
 
-## MultiQC
+If bambu is used:
+
+* `bambu/`  
+  * `extended_annotations.gtf` - a gtf file that contains both annotated and novel transcripts  
+  * `counts_gene.txt` - gene expression estimates
+  * `counts_transcript.txt` - transcript expression estimates
+
+If StringTie2 is used:
+
+* `stringtie2/`
+  * `*.bam`
+     Per-sample coordinate sorted alignment files in [`*.bam`](https://samtools.github.io/hts-specs/SAMv1.pdf) format.
+  * `*.stringtie.gtf`
+     Per-sample annotations for novel transcripts obtained in *StringTie2*.
+  * `stringtie.merged.gtf`
+     Extended annotation that combines provided gtf with gtf files from each sample via *StringTie2 Merge*.
+  * `counts_gene.txt` - gene expression estimates calculated by featureCounts.
+  * `counts_gene.txt.summary` - featureCounts gene level log file.
+  * `counts_transcript.txt` - transcript expression estimates calculated by featureCounts.
+  * `counts_transcript.txt.summary` - featureCounts transcript level log file.
+
+</details>
 
 *Documentation*:  
-[MultiQC](https://multiqc.info/docs/)
+[bambu](https://github.com/GoekeLab/bambu), [StringTie2](https://ccb.jhu.edu/software/stringtie/), [featureCounts](http://bioinf.wehi.edu.au/featureCounts/)
 
 *Description*:  
-*MultiQC* is a visualisation tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available within the report data directory.  
+After genomic alignment, novel transcripts can be reconstructed using tools such as bambu and StringTie2. Quantification can then be performed on a more complete annotation based on the transcripts detected within a given set of samples. bambu performs both the reconstruction and quantification steps. An an alternative approach, we also provides an option to run StringTie2 to identify novel transcripts. However, when multiple samples are provided, quantification for multiple samples are not implemented explicitly in the software. Hence a second step is required to merge novel transcripts across multiple samples followed by quantification for both gene and transcripts using featureCounts. You can skip transcript reconstruction and quantification by providing the `--skip_quantification` parameter.
```

Comment:
```suggestion
After genomic alignment, novel transcripts can be reconstructed using tools such as *bambu* and *StringTie2*. Quantification can then be performed on a more complete annotation based on the transcripts detected within a given set of samples. *bambu* performs both the reconstruction and quantification steps. An an alternative approach, we also provides an option to run *StringTie2* to identify novel transcripts. However, when multiple samples are provided, quantification for multiple samples are not implemented explicitly in the software. Hence a second step is required to merge novel transcripts across multiple samples followed by quantification for both gene and transcripts using *featureCounts*. You can skip transcript reconstruction and quantification by providing the `--skip_quantification` parameter.
```

Reply:
I don't think it matters as much if we leave these without emphasis. Note: we will also have to change this for all of the other tools in that document and possibly others too. Maybe we should leave things as they are?